airflow/example_dags/libs/helper.py

```diff
@@ -20,3 +20,11 @@
 
 def print_stuff():
     print("annotated!")
+
+def check_venv_dependency():
+    """Check if the virtualenv module is installed"""
+    try:
+        import virtualenv
+        return True
+    except ImportError:
+        return False
```

Comment:
You can use
`from airflow.operators.python import is_venv_installed`

https://github.com/apache/airflow/blob/8c0a4de66344647c759b6188cabafacf26ace9e3/airflow/operators/python.py#L66-L74